tests/runtest.py

```diff
@@ -852,6 +852,8 @@ def parse_argument():
     res.append(TestBase.TEST_UNSUPP_LANG)
     res.append(TestBase.TEST_SKIP)
 
+    if os.path.exists("failed-tests.txt"):
+        os.rename("failed-tests.txt", "failed-tests.txt.old")
     ftests = open("failed-tests.txt", "w")
```

Comment:
Sorry again, this might be better to create a variable and use it.
```suggestion
    failed_tests = "failed-tests.txt"
    if os.path.exists(failed_tests):
        os.rename(failed_tests, failed_tests + ".old")
    ftests = open(failed_tests, "w")
```